tests/func/test_get.py

```diff
@@ -87,3 +88,18 @@ def test_get_to_dir(dname, erepo):
 
     assert os.path.isdir(dname)
     assert filecmp.cmp(erepo.FOO, dst, shallow=False)
+
+
+def test_get_from_non_dvc_master(empty_dir, erepo, caplog):
```

Comment:
You don't seem to use `empty_dir`, is there really a point in creating a separate fixture for it? Seems like you could simply do your business right in this test.